account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -244,21 +245,21 @@ def _recompute_suspense_line(self, data, reconcile_auxiliary_id, manual_referenc
             if line["kind"] != "suspense":
                 new_data.append(line)
                 total_amount += line["amount"]
-                if line.get("currency_amount"):
-                    currency_amount += (
-                        self.env["res.currency"]
-                        .browse(line["line_currency_id"])
-                        ._convert(
-                            line["currency_amount"],
-                            self._get_reconcile_currency(),
-                            self.company_id,
-                            self.date,
-                        )
-                    )
+                # case of statement line with foreign_currency
+                if (
+                    line["kind"] == "liquidity"
+                    and line["line_currency_id"] != suspense_currency.id
+                ):
+                    currency_amount += self.amount_currency
```

Comment:
In case where we have company currency in USD, but the journal/bank account is in EURO, and we make a payment of an invoice in a foreign different currency (like CNY), the liquidity line is in journal currency, euro, so the amount_currency is in euro, but the currency of the suspense line should be in CNY since we have a foreign currency = CNY.
To avoid a convertion between euro and CNY, we take directly the amount_currency of the statement line (or we will have issues of conversion)